app/components/template-viewer/TemplateViewer.jsx

```diff
@@ -12,19 +12,19 @@ class TemplateViewer extends React.Component {
 module.exports = TemplateViewer;
 
 class TemplateList extends React.Component {
-
-    constructor() {
-        this.state = {templates: []}
-        $.getJSON("blobs/templates.json", (data)=> {
-            this.setState({templates: data});
-        }.bind(this)).fail(function (error) {
-            console.log(error);
-        }.bind(this));
+    constructor(props) {
+      super(props);
+      this.state = {templates: []}
+
+      $.getJSON("blobs/templates.json", (data) => {
+        this.setState({templates: data});
+      }.bind(this)).fail((error) => {
+        console.log(error);
+      }.bind(this));
```

Comment:
Are the .bind(this)'s redundant based on our new knowledge about arrow functions?